File: ProgramParameters.java
public class ProgramParameters
{

    public byte layers;
    public int[] nodes;
    public byte divisbleBy;
    public byte alogirthmNumber;
    public int iterations;
    public float learningRate;
    public ProgramParameters(byte layers1, int[] nodes1, byte divisbleBy1, int iterations, float learningRate, byte algoNum)
    {
        this.alogirthmNumber = algoNum;
        this.layers = layers1;
        this.nodes = nodes1;
        this.divisbleBy = divisbleBy1;
        this.iterations = iterations;
        this.learningRate = learningRate;
    }
}

File: ProgramParameters.py

class ProgramParameters:
    layers = None;
    nodes = None;
    divisbleBy = None;
    alogirthmNumber = None;
    iterations = None;
    learningRate = None;
    def _init_(self,layers1, nodes1, divisbleBy1, iterations, learningRate, algoNum):
        self.alogirthmNumber = algoNum;
        self.layers = layers1;
        self.nodes = nodes1;
        self.divisbleBy = divisbleBy1;
        self.iterations = iterations;
        self.learningRate = learningRate;
        
        
        
        